src/main/java/mate/academy/dao/CinemaHallDao.java

```diff
@@ -10,4 +10,8 @@ public interface CinemaHallDao {
     Optional<CinemaHall> get(Long id);
 
     List<CinemaHall> getAll();
+
+    Boolean update(CinemaHall cinemaHall);
```

Comment:
Consider using the primitive `boolean` instead of `Boolean` for the `update` method's return type to avoid unnecessary boxing and potential nullability issues.